android/src/main/java/dev/robingenz/capacitorjs/plugins/firebase/auth/FirebaseAuthentication.java

```diff
@@ -33,6 +33,9 @@
     private AuthStateChangeListener authStateChangeListener;
 
     public static final String ERROR_SIGN_IN_FAILED = "signIn failed.";
+    public static final String ERROR_INVALID_CUSTOM_TOKEN = "invalid custom token.";
+    public static final String ERROR_CUSTOM_TOKEN_UNIMPLEMENTED =
+        "signInWithCustomToken only works on the native layer. Use the js-sdk directly for web layer custom auth https://firebase.google.com/docs/auth/web/custom-auth.";
```

Comment:
```suggestion
        "signInWithCustomToken cannot be used in combination with skipNativeAuth.";
```

I suggest that we keep the error message simple here and add a helpful description for it in the `definitions.ts` file.